x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_significant_events_view/index.tsx

```diff
@@ -144,7 +160,7 @@ export function StreamDetailSignificantEventsView({ definition, refreshDefinitio
       features={detectedFeatures}
       isLoading={isFeatureDetectionLoading}
       closeFlyout={() => {
-        abort();
+        cancelFeatureIdentification();
```

Comment:
should this cancellation only happen when explicitly clicking `Stop` ? it was quite unexpected to see the task cancelled when I closed the flyout